### ISSUE TITLE
Paper "Untie the Knots: An Efficient Data Augmentation Strategy for Long-Context Pre-Training in Language Models"

### ISSUE THREAD

**Issue by rgtjf** [LCLM-Horizon/A-Comprehensive-Survey-For-Long-Context-Language-Modeling#3]
Thank you for the insightful report!

Could you please add the paper to the list:

- **ArXiv**: https://arxiv.org/abs/2409.04774
- **TLDR**: "Untie the Knots" (UtK) is a data augmentation strategy that improves long-context learning in LLMs by shuffling document chunks. It boosts both accuracy and training efficiency without changing data mix in 7B and 72B parameter models.
- **GitHub**: https://github.com/rgtjf/Untie-the-Knots
